applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcaster.c

```diff
@@ -4,29 +4,34 @@
  * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
```

Comment:
For naming consistency, isn't it better that this file is named broadcast_source.c?

Reply:
And that the function names are broadcast_source_(...)

Reply:
Yeah., probably